Makefile

```diff
@@ -191,6 +191,8 @@ install: go.install
 
 check test: go.test
```

Comment:
i'm thinking that `go.lint` should be a dependency of check, so that when `make check` is run, it will not only execute tests but also lint.